components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,51 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { SkeletonMixin } from './skeleton-mixin.js';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends SkeletonMixin(superclass) {
+
+	static get properties() {
+		// TODO: remove reflect
```

Comment:
Reminder about this TODO. This can hopefully be reactive state properties instead?

Reply:
yup, I just did this for debugging so I could see properties in the dom